deepmd/pt/model/descriptor/se_t.py

```
@@ -723,6 +814,88 @@ def forward(
+        dmatrix = dmatrix.view(-1, self.nnei, 4)
+        dmatrix = dmatrix.to(dtype=self.prec)
+        nfnl = dmatrix.shape[0]
+        # pre-allocate a shape to pass jit
+        result = torch.zeros(
+            [nfnl, self.filter_neuron[-1]],
+            dtype=self.prec,
+            device=extended_coord.device,
+        )
+        # nfnl x nnei
+        exclude_mask = self.emask(nlist, extended_atype).view(nfnl, -1)
+        for embedding_idx, ll in enumerate(self.filter_layers.networks):
```

Ruff

Loop control variable `ll` not used within loop body

Rename unused `ll` to `_ll`

(B007)